lib/model/model.go

```diff
@@ -1407,9 +1406,9 @@ func (m *Model) ResumeDevice(device protocol.DeviceID) {
 
 func (m *Model) IsPaused(device protocol.DeviceID) bool {
 	m.pmut.Lock()
-	paused := m.devicePaused[device]
-	m.pmut.Unlock()
```

Comment:
We sometimes do this for very short methods, as `defer` has a certain cost and there is no great risk of forgetting the unlock in a one line method with a single point of return. In this case it's not a performance critical method though so either defer or not defer is fine, and the defer case is a teensy bit cleaner so 👍